packages/lesswrong/server/givingSeason/resolvers.ts

```diff
@@ -11,3 +18,49 @@ addGraphQLResolvers({
 });
 
 addGraphQLQuery("GivingSeason2024DonationTotal: Float!");
+
+defineFeedResolver<Date>({
+  name: "GivingSeasonTagFeed",
+  args: "tagId: String!",
+  cutoffTypeGraphQL: "Date",
+  resultTypesGraphQL: `
+    newPost: Post
+    newComment: Comment
+  `,
+  resolver: async ({limit = 3, cutoff, offset, args, context}: {
+    limit?: number,
+    cutoff?: Date,
+    offset?: number,
+    args: {tagId: string},
+    context: ResolverContext
+  }) => {
+    const {tagId} = args;
+    const relevantPostIds = await context.repos.posts.getViewablePostsIdsWithTag(tagId);
```

Comment:
There's currently not an easy way to get all the comments on posts that have a particular tag without writing some custom SQL, which we can't do here without a lot of work because the queries need to use the "feed query" architecture. To get around this we just fetch a list of all the post ids with the tag we're looking for and get all the comments on those posts. This method wouldn't scale well, but I expect it to be fine for the quantity of posts we'll get for single-week giving season events.